meson.build

```diff
@@ -171,8 +171,9 @@ src_libratbag = [
 	'src/driver-hidpp10.c',
 	'src/driver-logitech-g300.c',
 	'src/driver-logitech-g600.c',
-	'src/driver-roccat.c',
-	'src/driver-roccat-kone-pure.c',
+ 	'src/driver-roccat.c',
+ 	'src/driver-roccat-kone-pure.c',
```

Comment:
I don't know why, but the diffs are broken here.

Reply:
There was a weird new line, I think i fixed it in the new commit I think.